SPECS/azure-init/azure-init.spec

```diff
@@ -34,7 +34,7 @@ cargo build --all
 mkdir -p %{buildroot}%{_sharedstatedir}/azure-init
 mkdir -p %{buildroot}%{_unitdir}
 install -m 0755 target/debug/azure-init %{buildroot}%{_sharedstatedir}/azure-init/azure-init
-install -m 0644 config/azure-init.service %{buildroot}%{_unitdir}/azure-init.service
+install -m 0644 config/azure-provisioning-agent.service %{buildroot}%{_unitdir}/azure-provisioning-agent.service
```

Comment:
At this point we might as well install target this to `azure-init.service` since future version of the package will be using that anyway.